app/graphql/types/query_type/edition_type_or_subtype.rb

```diff
@@ -3,7 +3,7 @@
 module Types
   class QueryType
     class EditionTypeOrSubtype < Types::BaseUnion
-      EDITION_TYPES = [Types::EditionType, Types::RoleType, Types::WorldIndexType, Types::MinistersIndexType].freeze
+      EDITION_TYPES = [Types::EditionType].freeze
```

Comment:
**suggestion:** tidy up

If we do go down this route, I'd have thought this class/type resolution could be fully deleted. I don't think it's doing anything meaningful with the edition type descendants removed